kinto/tests/test_default_bucket.py

```diff
@@ -15,7 +15,7 @@ def test_default_bucket_exists_and_has_user_id(self):
         result = bucket.json
         settings = self.app.app.registry.settings
         hmac_secret = settings['cliquet.userid_hmac_secret']
-        bucket_id = hmac_digest(hmac_secret, self.principal)[:32]
+        bucket_id = '%s' % UUID(hmac_digest(hmac_secret, self.principal)[:32])
```

Comment:
Yes I was using ``'%s' % `` to be compatible with both Py3 and Py2 but ``text_type`` works  :)